src/test/kotlin/org/malachite/estella/util/DatabaseClearer.kt

```diff
@@ -0,0 +1,53 @@
+package org.malachite.estella.util
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.context.annotation.Bean
+import org.springframework.stereotype.Component
+import java.sql.Connection
+import java.sql.ResultSet
+import java.sql.Statement
+import javax.sql.DataSource
+
+@Component
+class DatabaseClearer {
```

Comment:
Czyli zgrubsza mamy komponent który odpala nam kod SQL do czyszczenia?

Reply:
Yup, ale i tak musisz go sobie gdzieś wywołać, więc jak ktoś go nie użyje w kodzie, to będzie git